src/data/hprobe.timer

```diff
@@ -5,6 +5,7 @@ ConditionPathExists=/etc/telemetrics/opt-in
 [Timer]
 OnCalendar=daily
 AccuracySec=12h
+RandomizedDelaySec=21600
```

Comment:
Revisiting the `systemd.timer` documentation, let me see if I understand correctly.

With `AccuracySec=12h` on its own, the hprobe.timer will trigger hprobe.service once per day at a fixed time of day, at a random time between `00:00` and `12:00`. However, combined with `RandomizedDelaySec=21600`, a random time between `00:00` and `06:00` is chosen after every trigger of hprobe.service and shifted by a fixed value between 0 and 12 hours (to account for `AccuracySec=12h`). The end result is the hprobe records will still be sent once per day, but will be spread over a wider range of time (an 18-hour range instead of a 12-hour range), and record creation is randomized during that window.

Does the above look correct? If so, IMO this is definitely an improvement.

(I think when I originally filed the feature request, the `AccuracySec=12h` setting wasn't working well, but that issue (or issues) may have been addressed in recent systemd releases.)